### DISCUSSION TITLE
UBL 10x10 and 15x15 dont measure all points

### DISCUSSION THREAD

**Discussion by KevinStuewe** [Jyers/Marlin#793] [Q&A] [answered]
Hey, 
i just flashed one of the precompiled new firmwares to my Ender 3 V2. 
First I used: E3V2-UBL-BLTouch-15x15-HS-v4.2.2.bin and after that the version with the 10x10 grid.
I let them create their grids. 
But the 15x15 just measured around 170/225 points and the 10x10 measured around 57/100. 
Then it displayed "Probed all reachable points"

The view mesh option showed the 15x15 and 10x10 grids even though not all points were measured.
Am I getting something wrong? Cause I guess thats not how I think it is supposed to work.

**Comment by Tuncay-Ayhan** [comment, marked as answer]
Read this. https://github.com/Jyers/Marlin/issues/672

**Reply by KevinStuewe** [reply]
Thanks
